src/main/java/software/amazon/nio/spi/s3/S3XFileSystemProvider.java

```diff
@@ -14,34 +14,16 @@
  * limitations under the License.
  */
 
-package software.amazon.nio.spi.s3x;
+package software.amazon.nio.spi.s3;
```

Comment:
You seem to have moved some of the s3x... classes into the s3 package. Is this intentional? If so you should probably move the rest as well?

Reply:
Yes it was intentional. By changing the visibility of `fileSystemInfo` in the superclass of the `S3XFileSystemProvider`, I needed to move this one to the same package.

The one left in `s3x` package is `S3XFileSystemInfo`, I can move it👍

Reply:
Done in 1c80df9